Funcky.Test/Extensions/FunctionCompositionTest.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+using FsCheck;
+using FsCheck.Xunit;
+using Funcky.Extensions;
+using Xunit;
+
+namespace Funcky.Test.Extensions
+{
+    public sealed class FunctionCompositionTest
+    {
+        [Property]
+        public Property FunctionWithTheSameInputAsOutputCanBeComposedBothWays(int argument)
+        {
+            Func<int, int> f = x => x + 2;
```

Comment:
would this also work with local functions?

Reply:
The normal parameters can be a local function, but the extension function is implemented on Func so you would need to cast the local function to a Func first.